### PR TITLE
Replace prettytable with comfy_table

### PR DESCRIPTION
**Pull request by jmg-duarte**
Table looks _slightly_ different, not that we can't change it, but letting you guys know
```
┌─────────────────────────────────────────────┬───────────┬───────────────┬────────┬────────┐
│ NODE ID                                     ┆ HOST      ┆ TOPIC         ┆ SIZE   ┆ ACTIVE │
╞═════════════════════════════════════════════╪═══════════╪═══════════════╪════════╪════════╡
│ blDwxEOHbf/NqhNza6aBiG2qCdH.J5q429X9YCbnFTY ┆ localhost ┆ default-topic ┆ 934432 ┆ *      │
└─────────────────────────────────────────────┴───────────┴───────────────┴────────┴────────┘
```

### CODE REVIEW

File: tags.py
Comment (rkuhn) [inline tags.py:1]: sneaky :-)

**Review by rkuhn — APPROVED** [review]

**Comment by rkuhn** [comment]
I don’t mind the slightly different look.

File: tags.py
Comment (jmg-duarte) [reply tags.py:1]: Oops that's just from my bad habit of adding everything (I have aliases, so `gaa` and commit 😅)

File: rust/actyx/ax/src/cmd/mod.rs [hunk 1]
```diff
@@ -59,19 +59,19 @@ impl ConsoleOpt {
...
-            .column_separator('│')
-            .borders('│')
-            .separators(&[LinePosition::Top], LineSeparator::new('─', '┬', '┌', '┐'))
-            .separators(&[LinePosition::Title], LineSeparator::new('─', '┼', '├', '┤'))
-            .separators(&[LinePosition::Bottom], LineSeparator::new('─', '┴', '└', '┘'))
-            .padding(1, 1)
-            .build();
-    }
-}
+// pub(crate) mod consts {
```
Comment (Kelerchian) [inline rust/actyx/ax/src/cmd/mod.rs:62]: Is this kept for, in the future, referring back to old table style?

**Review by Kelerchian — APPROVED** [review]
No concern on the style change

File: rust/actyx/ax/src/cmd/mod.rs (hunk 1 above)
Comment (jmg-duarte) [reply rust/actyx/ax/src/cmd/mod.rs:62]: Nope, because I forgot to delete. Nice catch

**Pull request merged**